Cargo.toml

```diff
@@ -25,7 +25,7 @@ prost-codec = ["raft-proto/prost-codec"]
 # Make sure to synchronize updates with Harness.
 [dependencies]
 log = ">0.2"
-protobuf = "2"
+protobuf = { git = "https://github.com/nrc/rust-protobuf", branch = "v2.8" }
```

Comment:
This unfortunately renders Raft unpublishable, if we merge this it will require a future cleanup PR.